tools/ci

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
To ensure cross-platform compatability, this should probably be a rust-script

i.e. using the xtask pattern

Reply:
yeah that does seem like a reasonable approach / would allow ci and contributors to use the exact same command on all platforms.

https://github.com/matklad/cargo-xtask

The only downsides that i can see:

1. more boilerplate (the consistency is worth it imo)
2. requires adding a default `.cargo/config` file, which will conflict with our recommendation to rename `.cargo/config_fast_builds` to `.cargo/config`.

Reply:
xtask is a big enough change that i think its probably worth doing in a separate pr